browser/mapping.go

```diff
@@ -1042,3 +1017,23 @@ func panicIfFatalError(ctx context.Context, err error) {
 		k6ext.Abort(ctx, err.Error())
 	}
 }
+
+// exportArg exports the value and returns it.
+// It returns nil if the value is undefined or null.
+func exportArg(gv goja.Value) any {
+	if gv == nil || gv == goja.Undefined() || gv == goja.Null() {
```

Comment:
Not important, but wondering if exporting `gojaValueExists` could be useful? Possibly in another PR.

Reply:
Copying it here would be more useful :) It's a simple function. I can do that now. WDYT?

Reply:
I understand that a little copying is sometimes better than reusing a method, but I don't understand why that must be the case for this, especially in the mapping layer which is going to be very long and difficult to navigate if we keep adding functions into this file.

Reply:
On the other hand having a function closer to the caller does indeed make things simpler.

Since the mapping layer will be focused on the goja parsing, would it eventually make sense for the parsing logic to live in the `browser` package instead of the `common` package? If that's the eventual goal then I can see how copying `gojaValueExists` into `mapping.go` would be a good step forward. That could also mean that we refactor goja stuff into the mapping layer, we should also be refactoring the parse code from the `common` package too. WDYT?

Reply:
> I don't understand why that must be the case for this

Because the `gojaValueExists` is a little function. Reusing it doesn't add any value. However, exporting the function pollutes the API surface area.

> On the other hand having a function closer to the caller does indeed make things simpler.

+1

> would it eventually make sense for the parsing logic to live in the browser package instead of the common package?

Exactly: https://github.com/grafana/xk6-browser/issues/1064

> That could also mean that when we refactor goja stuff into the mapping layer, we could also be moving the parse code from the common package too.

I'm all in!

Reply:
> Because the gojaValueExists is a little function (more on [this here](https://www.youtube.com/watch?v=PAAkCSZUG1c&t=568s)). Reusing it doesn't add any value. However, exporting the function pollutes the API surface area.

Sure, happy with the idea 👍

Reply:
@ankur22 Done: https://github.com/grafana/xk6-browser/pull/1205/commits/43be12cbba98953c37e5b043238a80f177c3e412, https://github.com/grafana/xk6-browser/pull/1205/commits/f5f26e93eedf7702ca0a68aed6859b8b5e5bab38, https://github.com/grafana/xk6-browser/pull/1205/commits/36a214ecbf2e038fb2f106f436174c85662a02a0.